.github/workflows/test.yml

```
@@ -44,11 +39,18 @@ jobs:
         run: |
           python -m nox --error-on-missing-interpreters -s tests-${{ matrix.python_version }}
         shell: bash
-        if: matrix.python_version != 'pypy2'
+        if: ${{ ! startsWith( matrix.python_version, 'pypy' ) }}
 
       # Binary is named 'pypy', but setup-python specifies it as 'pypy2'.
       - name: Run nox for pypy2
         run: |
           python -m nox --error-on-missing-interpreters -s tests-pypy
         shell: bash
         if: matrix.python_version == 'pypy2'
+
+      # Binary is named 'pypy3.7', but setup-python specifies it as 'pypy-3.7'.
+      - name: Run nox for pypy3
+        run: |
+          python -m nox --error-on-missing-interpreters -s tests-pypy3.7
```

```suggestion
          python -m nox --error-on-missing-interpreters -s tests-pypy3
```

That'll probably make this work; if this still fails for some reason?